tests/FormAccessibleTest.php

```diff
@@ -40,8 +40,8 @@ public function testItCanMutateValuesForForms()
         $model = new ModelThatUsesForms($this->modelData);
         $this->formBuilder->setModel($model);
 
-        $this->assertEquals($model->getFormValue('string'), 'ponmlkjihgfedcba');
-        $this->assertEquals($model->getFormValue('created_at'), $this->now->timestamp);
+        $this->assertEquals($model->getFormValue('string'), 'PONMLKJIHGFEDCBA');
```

Comment:
This is expected, because before the PR the `Model::getAttributeFromArray()` will simply get the attribute value from the `$attributes` array on the eloquent `Model`.

But when using `Model::getAttribute()` it will call any eloquent attribute mutator methods which in our case `ModelThatUsesForms::getStringAttribute()`

Using `Model::getAttribute()` will trigger any eloquent logic, casting ($dates, $casts), attribute mutation ...